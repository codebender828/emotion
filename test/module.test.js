import { setupTest, get } from '@nuxt/test-utils'

describe('module', () => {
  setupTest({
    testDir: __dirname,
    fixture: '../example',
    server: true
  })

  test('/ (pink)', async () => {
    const html = await get('/').then(r => r.body)
    expect(html).toMatchSnapshot()
  })

  test('/ (normal)', async () => {
    const html = await get('/normal').then(r => r.body)
    expect(html).toMatchSnapshot()
  })
})
