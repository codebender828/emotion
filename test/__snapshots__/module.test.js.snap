// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic / (normal) 1`] = `
"<!doctype html>
<html data-n-head-ssr data-n-head=\\"\\">
  <head data-n-head=\\"\\">
    <title data-n-head=\\"true\\"></title><style data-vue-ssr-id=\\"17cfdfa9:0\\">.nuxt-progress{position:fixed;top:0;left:0;right:0;height:2px;width:0;opacity:1;transition:width .1s,opacity .4s;background-color:#000;z-index:999999}.nuxt-progress.nuxt-progress-notransition{transition:none}.nuxt-progress-failed{background-color:red}</style><style>@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');html,body{font-family:'Source Sans Pro',sans-serif;color:#0a1f44;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;min-height:100vh;text-align:center;}a{padding:.5em;border:1px solid grey;border-radius:.3em;color:#0a1f44;-webkit-text-decoration:none;text-decoration:none;}.css-9ehno9{color:#42b883;}.css-amiok5{font-size:60px;}</style><script>window.$emotionSSRIds = [\\"1qjmn0x\\",\\"9ehno9\\",\\"amiok5\\"]</script>
  </head>
  <body data-n-head=\\"\\">
    <div data-server-rendered=\\"true\\" id=\\"__nuxt\\"><!----><div id=\\"__layout\\"><div><div><h1 class=\\"css-amiok5\\">
    Nuxt
    <span class=\\"css-9ehno9\\">
      JS
    </span></h1> <h1>
    The
    <span class=\\"css-9ehno9\\">
      Vue.js
    </span>
    Framework
  </h1></div> <a href=\\"/\\" class=\\"nuxt-link-active\\">
    Go Pink
  </a></div></div></div><script>window.__NUXT__={layout:\\"default\\",data:[{}],error:null,serverRendered:true};</script><script src=\\"/_nuxt/69c7edc4208cfaa53317.js\\" defer></script><script src=\\"/_nuxt/fb29fc0167d1342be629.js\\" defer></script><script src=\\"/_nuxt/9de0db02460713695069.js\\" defer></script><script src=\\"/_nuxt/21afeb84309963f6c07e.js\\" defer></script><script src=\\"/_nuxt/da65cb3c212d7ca40786.js\\" defer></script>
  </body>
</html>
"
`;

exports[`basic / (pink) 1`] = `
"<!doctype html>
<html data-n-head-ssr data-n-head=\\"\\">
  <head data-n-head=\\"\\">
    <title data-n-head=\\"true\\"></title><style data-vue-ssr-id=\\"17cfdfa9:0\\">.nuxt-progress{position:fixed;top:0;left:0;right:0;height:2px;width:0;opacity:1;transition:width .1s,opacity .4s;background-color:#000;z-index:999999}.nuxt-progress.nuxt-progress-notransition{transition:none}.nuxt-progress-failed{background-color:red}</style><style>@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');html,body{font-family:'Source Sans Pro',sans-serif;color:#0a1f44;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;min-height:100vh;text-align:center;}a{padding:.5em;border:1px solid grey;border-radius:.3em;color:#0a1f44;-webkit-text-decoration:none;text-decoration:none;}.css-9ehno9{color:#42b883;}.css-amiok5{font-size:60px;}.css-1lrxbo5{color:hotpink;}</style><script>window.$emotionSSRIds = [\\"1qjmn0x\\",\\"9ehno9\\",\\"amiok5\\",\\"1lrxbo5\\"]</script>
  </head>
  <body data-n-head=\\"\\">
    <div data-server-rendered=\\"true\\" id=\\"__nuxt\\"><!----><div id=\\"__layout\\"><div class=\\"css-1lrxbo5\\"><div><h1 class=\\"css-amiok5\\">
    Nuxt
    <span class=\\"css-9ehno9\\">
      JS
    </span></h1> <h1>
    The
    <span class=\\"css-9ehno9\\">
      Vue.js
    </span>
    Framework
  </h1></div> <a href=\\"/normal\\">
    Go Normal
  </a></div></div></div><script>window.__NUXT__={layout:\\"default\\",data:[{}],error:null,serverRendered:true};</script><script src=\\"/_nuxt/69c7edc4208cfaa53317.js\\" defer></script><script src=\\"/_nuxt/4a6e9f26a8370c85a917.js\\" defer></script><script src=\\"/_nuxt/9de0db02460713695069.js\\" defer></script><script src=\\"/_nuxt/21afeb84309963f6c07e.js\\" defer></script><script src=\\"/_nuxt/da65cb3c212d7ca40786.js\\" defer></script>
  </body>
</html>
"
`;
